model/generate.go

```diff
@@ -261,12 +265,12 @@ func (g *GeneratedProject) saveNewBuildsAndTasks(ctx context.Context, cachedProj
 		syncAtEndOpts = patchDoc.SyncAtEndOpts
 	}
 
-	tasksInExistingBuilds, err := AddNewTasks(ctx, true, v, p, newTVPairsForExistingVariants, syncAtEndOpts, g.TaskID)
+	tasksInExistingBuilds, err := AddNewTasks(ctx, false, v, p, newTVPairsForExistingVariants, syncAtEndOpts, g.TaskID)
```

Comment:
What was the rational for changing this to false?

Reply:
Just that it doesn't do anything with us setting the build's activated status in the next line. Both ways are probably slightly confusing

Reply:
Hmm we can talk about this in person if that's easier--Are we certain that activating builds in AddNewBuilds is enough? Should AddNewTasks also take in this "variantsToActivate" and use that [here](https://github.com/evergreen-ci/evergreen/blob/master/model/lifecycle.go#L1373)? Or if AddNewBuilds is enough should we remove that parameter from AddNewTasks? I just want to check since generate tasks bugs are weird haha, at the least a comment might be good if you think switching it back to true will cause a problem